db/migrate/20170905153238_create_order_table.rb

```diff
@@ -0,0 +1,10 @@
+class CreateOrderTable < ActiveRecord::Migration
+  def change
+    create_table :orders do |t|
+      t.string :user_email
+      t.integer :route, null:false, :references => [:routes, :id]
```

Comment:
1) indentation
2) use new style of key/value pairs `key: value`

Reply:
:+1: